ddtrace/propagation/utils.py

```diff
@@ -1,3 +1,6 @@
+from ddtrace.utils.cache import cached
+
+
 def get_wsgi_header(header):
```

Comment:
`@cached()` ?

Reply:
there was no sign of `get_wsgi_header` in profiling data but I'll see what happens if this is also cached. If there are no measurable benefits I'll leave it uncached to save a bit of memory.

Reply:
👍🏻 no strong opinion, but I don't see why we wouldn't get similar benefits... I suppose it depends on how often it is called

maybe for Django it is low number of calls, but WSGI wrappers maybe it is much higher usage?